package.json

```diff
@@ -44,6 +44,7 @@
     "nsp": "^2.6.3",
     "pre-commit": "^1.2.2",
     "standard-version": "^4.0.0",
+    "webpack": "^2.4.1",
```

Comment:
`peerDependency` will be sufficient here. It's acceptable to require a developer to install webpack as a part of the devEnv setup. 

For CI, the Travis build runs off a matrix which installs Webpack to test against including multiple versions for canary / major release beta testing as necessary.